docs/connected-ZTP-flow-hub-deployment.md

```diff
@@ -296,7 +296,19 @@ spec:
     name: assisted-deployment-pull-secret
 ```
 
-- **NMState Config**: This is an **optional** configuration that you want to add when the Network configuration needs some adjustments like work with Bonding or use a concrete VLAN or just declaring a Static IP. The NMState it's a generic/standard configuration that could be used in a separate way of Assisted Installer/ACM and the [documentation can be found here](https://github.com/nmstate/nmstate) and [here are some examples](https://nmstate.io/examples.html). One NMState profile will map in a relation of 1-1 to an InfraEnv (we will cover this one later) and this profile should cover all nodes involved on the cluster.
+- **NMState Config**: This is an **optional** configuration that you want to add when the Network configuration needs some adjustments like work with Bonding or use a concrete VLAN or just declaring a Static IP. The NMState is a generic/standard configuration that could be used in a separate way of Assisted Installer/ACM and the [documentation can be found here](https://github.com/nmstate/nmstate) and [here are some examples](https://nmstate.io/examples.html). 
+
+Each NMStateConfig is labeled with a key:value label, and this label will be set also in the InfraEnv (check the next step) object used to configure this cluster.
```

Comment:
Each NMStateConfig manifests are labeled with a key:value entries which it will be refereced from the Infraenv (check the next step) object used to configure this cluster.